dockers/docker-auditd/config_checker.py

```diff
@@ -31,76 +44,122 @@ def run_command(cmd):
     output, error = p.communicate()
     if p.returncode == 0:
         return p.returncode, output
+    logger.log_error("Command failed: {} (Return code: {})".format(cmd, p.returncode))
+    logger.log_error("Error: {}".format(error))
     return p.returncode, error
 
 
 def get_hwsku():
-    with open("/etc/sonic/config_db.json") as fp:
-        config_db = json.load(fp)
-    if "DEVICE_METADATA" in config_db and "hwsku" in config_db["DEVICE_METADATA"]["localhost"]:
-        return str(config_db['DEVICE_METADATA']['localhost']['hwsku'])
+    try:
+        with open("/etc/sonic/config_db.json") as fp:
```

Comment:
Seems parsing json file directly does not work for multi-asic. Suggest use `sonic-cfggen -d -v DEVICE_METADATA.localhost.hwsku`.